backend/core/src/listings/listings.service.ts

```diff
@@ -177,6 +178,75 @@ export class ListingsService {
     return result
   }
 
+  async rawListWithFlagged() {
+    const userAccess = await this.userRepository
+      .createQueryBuilder("user")
+      .select("user.id")
+      .leftJoin("user.roles", "userRole")
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+      .where("user.id = :id", { id: (this.req.user as User)?.id })
+      .andWhere("userRole.is_admin = :is_admin", { is_admin: true })
```

Comment:
Found another bug. Since this is just `is_admin` if you are a jurisdictional admin the button is there but the call to the backend fails